package.json

```diff
@@ -10,13 +11,28 @@
     "react-dom": "^18.2.0",
     "react-redux": "^8.1.3",
     "react-router-dom": "^6.16.0",
+    "react-scripts": "^5.0.1",
     "redux": "^4.2.1",
     "redux-thunk": "^2.4.2"
   },
   "devDependencies": {
     "@babel/core": "^7.23.3",
     "@babel/preset-env": "^7.23.3",
     "@babel/preset-react": "^7.23.3",
-    "babel-loader": "^9.1.3"
+    "babel-loader": "^9.1.3",
+    "webpack": "^5.89.0",
+    "webpack-cli": "^5.1.4"
```

Comment:
- [ ] Great work installing `webpack`. However, I noticed that your application still shows a blank screen as highlighted by the previous reviewer. 

![image](https://github.com/Gardimy/Set_up_hello_project_webpack/assets/77106962/63a88e58-b5e5-42d0-8552-8625fea2f208)

---
The above is the error log shown in the terminal. I would advise you to follow this [tutorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a:~:text=com/es6%2Dmodule-,Webpack,-At%20its%20core) step by step to properly set up this project. 👍 😎 

---
Also note that your **_entry point_** in the _**webpack.config.js**_ file says `'./app/javascript/application.js'` and not `'./app/javascript/component/application.js'`. 💻 

![image](https://github.com/Gardimy/Set_up_hello_project_webpack/assets/77106962/9cab2913-bb73-4cae-8cc2-68f4478f4980)